src/components/Review/DecisionPreview/ReviewPreviewModal.tsx

```diff
@@ -44,7 +44,7 @@ const ReviewPreviewModal: React.FC<PreviewProps> = ({
     // generates new Previews every time it's opened
     <Modal id="preview-modal" open={open} closeOnDimmerClick={false}>
       <Modal.Header>{strings.REVIEW_DECISION_PREVIEW_HEADER}</Modal.Header>
-      <Modal.Content>
+      <Modal.Content scrolling>
```

Comment:
Fixes the overflow problem noticed by @nmadruga when content was too long. This makes it scroll within the modal.